### ISSUE TITLE
[Bug] Map is not loading in a custom React application

### ISSUE THREAD

**Issue by zmeecer** [keplergl/kepler.gl#1648]
**Describe the bug**

Create an application via create-react-app, integrate kepler.gl following an example from Readme. The side panel is visible, but there is no map at all. In DevTools: no errors, no requests to Mapbox. 

**Expected behavior**

The same as on the demo site: side panel and the map

**Screenshots**

<img width="766" alt="Screenshot 2021-10-29 at 14 03 37" src="https://user-images.githubusercontent.com/1756469/139435194-db2ec792-a7a1-4c45-a7b5-b8fc37b0810b.png">

**Desktop (please complete the following information):**
 - OS: MacOS Big Sur
 - Browser: Chrome v95

**Comment by Pipazoul** [comment]
How have you passed your mapbox token ?

**Comment by zmeecer** [comment]
> How have you passed your mapbox token ?

yep, tried the same token in a minimal `react-map-gl` example - works fine.

in some cases I get these warnings, what can be relevant to the issue:
<img width="829" alt="Screenshot 2021-10-31 at 19 35 35" src="https://user-images.githubusercontent.com/1756469/139597044-23898365-c20c-428e-a0d8-700934ccaa25.png">

and here is my code:
```jsx
const Map = () => {
  const dispatch = useDispatch();

  useEffect(() => {
    dispatch(addDataToMap(defaultData));
  }, [dispatch]);

  return (
    <KeplerGl
      id="foo"
      mapboxApiAccessToken="$token$"
      width={800}
      height={600}
    />
  );
};
```

**Comment by zmeecer** [comment]
An interesting fact: the demo application is working correctly - with the same set of dependencies. Now trying to make my application and the demo one as similar as possible. I will provide any further findings

**Comment by BugsBunny338** [comment]
I just ran into the same problem, except I cannot follow @hmzls's advice as react-palm does not support React v17 that I'm currently running 😬 Any ideas how to fix this?

**Comment by shashank0092** [comment]
i am also facing same issue please anyone help

**Comment by Karthigeyankiruba** [comment]
> I just ran into the same problem, except I cannot follow @hmzls's advice as react-palm does not support React v17 that I'm currently running 😬 Any ideas how to fix this?

got any solutions for this issue?
